File: Lab_05(k).c
#include<stdio.h>


int main()
{
    int i,f=1,num;
    printf("Enter a number to calculate factorial using for loop: ");
    scanf("%d",&num);
    for(i=1; i<=num; i++)
        f=f*i;
    printf("Factorial of %d is: %d",num,f);

    printf("\n Enter a number to calculate factorial using while loop: ");

    scanf("%d",&num);
      while(i<=num)
    {
        f=f*i;
        i++;
    }

    printf("\n The Factorial of %d is %d:",num,f);
}

File: Lab_05(k).py


num = int(input("Enter a number to check For loop : "))

fac = 1

for i in range(1, num + 1):
    fac = fac * i

print("factorial of ", num, " is ", fac)


num = int(input("\n Enter a number to check while loop: "))

fac = 1
i = 1

while i <= num:
    fac = fac * i
    i = i + 1

print("factorial of ", num, " is ", fac)